src/server/scripts/Northrend/IcecrownCitadel/boss_sindragosa.cpp

```diff
@@ -1549,7 +1549,8 @@ class spell_frostwarden_handler_focus_fire : public SpellScriptLoader
                 PreventDefaultAction();
                 if (Unit* caster = GetCaster())
                 {
-                    caster->AddThreat(GetTarget(), -float(GetSpellInfo()->GetEffect(caster->GetMap()->GetDifficultyID(), EFFECT_1)->CalcValue()));
+                    if (SpellEffectInfo const* effect = GetSpellInfo()->GetEffect(target->GetMap()->GetDifficultyID(), EFFECT_1))
```

Comment:
`if (SpellEffectInfo const* effect = GetSpellInfo()->GetEffect(caster->GetMap()->GetDifficultyID(), EFFECT_1))`

Reply:
Yes, travis-ci points out `use of undeclared identifier 'target'` which indicates that it should be `caster` like before.